src/main/bash/sdkman-availability.sh

```diff
@@ -16,10 +16,10 @@
 #   limitations under the License.
 #
 
-function __sdkman_update_broadcast_and_service_availability() {
+function __sdkman_update_service_availability() {
 	local broadcast_live_id=$(__sdkman_determine_broadcast_id)
 	__sdkman_set_availability "$broadcast_live_id"
-	__sdkman_update_broadcast "$broadcast_live_id"
+	__sdkman_update_broadcast_id "$broadcast_live_id"
```

Comment:
This is still needed for the service availability check but will also go away in time. We use the hardcoded nginx `/broadcast/id` mapping here.